spec/features/ncr/work_orders/create_spec.rb

```diff
@@ -112,6 +112,17 @@
       expect(page).to_not have_content("Approving official email can't be blank")
     end
 
+    scenario "inactive users do not appear as potential approvers", :js do
```

Comment:
there is a test for this here: https://github.com/18F/C2/pull/734/files#diff-9965b424c2f291874f17f5d65d09fc9dR1

if that was not a good / obvious place for it, should we perhaps remove it alongside adding this?